var.tfvars

```diff
@@ -78,3 +78,17 @@ cluster_id                  = ""
 #upgrade_channel            = ""  #(stable-4.x, fast-4.x, candidate-4.x) eg. stable-4.5
 #upgrade_pause_time         = "90"
 #upgrade_delay_time         = "600"
+## Uncomment any one of the below formats to use external proxy. Default 'port' will be 3128 if not specified. Not authenticated if 'user' is not specified.
+#proxy = {}
+#proxy = {server = "hostname_or_ip"}
+#proxy = {server = "hostname_or_ip", port = "3128", user = "pxuser", password = "pxpassword"}
```

Comment:
Why the proxy variable changes in the PR for image upgrade ?

Reply:
Code had got merged during rebase command, updated now with latest commit.